src/components/Send/ShowUtxos.tsx

```diff
@@ -160,7 +160,7 @@ const UtxoRow = memo(
               type="checkbox"
               checked={checked}
               onChange={() => {
-                onToggle && onToggle(utxoIndex, isFrozen)
```

Comment:
what will happen to `isFrozen` that we're not getting from above, how would you handle frozen utxo and non-frozen utxo?

Reply:
```utxo``` will contain all the properties of the particularly selected utxo. So to access ```isFrozen``` property we can use ```utxo.isFrozen```. Basically we are trying to access whole utxo instead of just one of its property ```isFrozen```.